solutions/swb-app/src/schemas/projects/getProjectSchema.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// Schema for GetProject API
+import { Schema } from 'jsonschema';
+
+const GetProjectSchema: Schema = {
+  id: '/getProject',
+  type: 'object',
+  properties: {
+    user: { type: 'AuthenticatedUser' },
```

Comment:
How does this type work? You should describe schema (see details: https://stackoverflow.com/a/62585173)

Reply:
All I know is that it does seem to work for our current usage. When I pass in `res.locals.user` in the route, it is accepted as an `AuthenticatedUser` and the `validate` method passes.

Reply:
Interesting. Could you try to set random object, e.g. `res.locals.user = { a: 1, b: 2 }`? Wonder if it works